src/gridDynMain/CMakeLists.txt

```diff
@@ -20,8 +20,13 @@ add_executable(griddynMain ${griddynMain_sources} ${griddynMain_headers})
 set(GRIDYNMAIN_BINARY_LOC ${CMAKE_CURRENT_BINARY_DIR} PARENT_SCOPE)
 
 message(STATUS "execOnly:${executable_only_components}")
+
 target_link_libraries(griddynMain ${executable_only_components} runner)
 
+if(DIME_ENABLE)
+    target_link_libraries(griddynMain networking)
+endif(DIME_ENABLE)
+
```

Comment:
the networking library should be linked in by way of the runner library, which should include the optional components.  Eventually we will want to partition the dimerunner into its own library since it will be dependent on the the runner, but that is a couple steps down the road.